README-KO.md

```diff
@@ -262,7 +262,7 @@ Scala는 매우 강력하며 여러가지 페러다임에 적용 가능한 언
   }
   ```
 
-- 함수 호출자는 반드시 함수의 정의를 따라야 합니다. 예를 들어, 함수가 괄호 없이 선언되었다면 괄호 없이 호출되어야 합니다. 이 것은 단지 문법적인 문제일 뿐만 아니라 `apply`를 호출 할 때에도 문제가 될 수 있습니다.
+- 메소드 호출자는 반드시 메소드의 정의를 따라야 합니다. 예를 들어, 메소드가 괄호 없이 선언되었다면 괄호 없이 호출되어야 합니다. 이 것은 단지 문법적인 문제일 뿐만 아니라 `apply`를 호출 할 때에도 문제가 될 수 있습니다.
```

Comment:
Callsite should follow method declaration, i.e. if a method is declared with parentheses, call with parentheses. Note that this is not just syntactic. It can affect correctness when apply is defined in the return object.

Reply:
Previous translation changed the original guide from 'with parenthesis' to 'without parenthesis'. As the following phrase is corresponds to the leading phrase, it does not affect the meaning, but I add the original guide just in case.